AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -305,18 +312,18 @@ async function networkCheckConnectionString(propertyName, connectionString, conn
                     `Could not access the endpoint "${hostPort.HostName}:${hostPort.Port}" configured in App Setting "${propertyName}".`;
                 subChecks.push({ title: title, level: maxCheckLevel, subChecks: connectivityCheckResult });
             } else { // Unsupported or invalid connection string format
-                var title = `Unable to parse the connection string configured in the App Setting "${propertyName}".  It is either not supported by this troubleshooter or invalid.`;
-                if (failureDetailsMarkdown != undefined) {
-                    subChecks.push({ title: title, level: 2, detailsMarkdown: failureDetailsMarkdown });
-                } else {
-                    subChecks.push({ title: title, level: 2 });
-                }
+               
+            var title = `Unable to parse the connection string configured in the App Setting "${propertyName}".  It is either not supported by this troubleshooter or invalid.`;
+            if (failureDetailsMarkdown != undefined) {
+                subChecks.push({ title: title, level: 2, detailsMarkdown: failureDetailsMarkdown });
+            } else {
+                subChecks.push({ title: title, level: 2 });
+            }       
+                
             }
-        }
-    } else {
-        var res = await networkCheckKeyVaultReferenceAsync(propertyName, connectionString, dnsServers, diagProvider, isVnetIntegrated);
-        res.forEach(item => subChecks.push(item));
-    }
+         }
+        
+      
     return subChecks;
 }
 

```

Comment:
This method is no longer used anywhere from what I can tell.  Can you confirm and delete it?  We now handle Key Vault References in DaaS.